hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/command/ws/server/AbstractGatewayWebSocket.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.cmdgw.command.ws.server;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+import javax.websocket.Session;
+
+import org.hawkular.accounts.websocket.Authenticator;
+import org.hawkular.accounts.websocket.WebsocketAuthenticationException;
+import org.hawkular.bus.common.BasicMessage;
+import org.hawkular.bus.common.BasicMessageWithExtraData;
+import org.hawkular.bus.common.BinaryData;
+import org.hawkular.cmdgw.MsgLogger;
+import org.hawkular.cmdgw.annotations.Feed;
+import org.hawkular.cmdgw.annotations.Ui;
+import org.hawkular.cmdgw.annotations.Ws;
+import org.hawkular.cmdgw.api.ApiDeserializer;
+import org.hawkular.cmdgw.api.AuthMessage;
+import org.hawkular.cmdgw.api.Authentication;
+import org.hawkular.cmdgw.api.UiSessionOrigin;
+import org.hawkular.cmdgw.command.Command;
+import org.hawkular.cmdgw.command.CommandContext;
+import org.hawkular.cmdgw.command.CommandContextFactory;
+import org.hawkular.cmdgw.command.Commands;
+import org.hawkular.cmdgw.command.NoCommandForMessageException;
+
+/**
+ * @author <a href="https://github.com/ppalaga">Peter Palaga</a>
+ */
+public abstract class AbstractGatewayWebSocket {
+    /**
+     * @param response
```

Comment:
I don't. It just gets in the way. But these empty javadocs are meaningless anyway and just clutter the code, so we should either fill them in or remove them.